index.js

```diff
@@ -256,6 +263,20 @@ var handleCloudWatch = function(event, context) {
   } else if (message.NewStateValue === "OK") {
       color = "good";
   }
+  
+  for (var i=0; i< accountIdMap.length; i++) {
+    if (accountIdMap[i].accountId === accountId) {
+      accountName = accountIdMap[i].name;
+      break;
+    }
+  }
+
+  for (var i=0; i < dimensions.length; i++) {
+    dimensionsText += dimensions[i].name + '=' + dimensions[i].value;
```

Comment:
Likewise this could be more expressive by using `forEach()` or `map()`:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map